### PR TITLE
Fix/recognition mode value

### PR DESCRIPTION
**Pull request by franek-scandit**

### CODE REVIEW

File: src/ios/ScanditSDK.mm [hunk 1]
```diff
@@ -331,9 +331,13 @@ - (NSDictionary *)lowerCaseOptionsFromOptions:(NSDictionary *)options {
 
 - (SBSScanSettings *)updateScanSettings:(SBSScanSettings *)scanSettings withSettings:(NSDictionary *)settings {
     if ([scanSettings respondsToSelector:@selector(setRecognitionMode:)]) {
-        NSString *recognitionMode = settings[@"recognitionMode"];
+        id recognitionMode = settings[@"recognitionMode"];
         if (recognitionMode != nil && [recognitionMode isKindOfClass:[NSString class]] && [recognitionMode isEqualToString:@"text"]) {
             scanSettings.recognitionMode = SBSRecognitionModeText;
+        } else if (recognitionMode != nil && [recognitionMode isKindOfClass:[NSString class]]) {
```
Comment (petradonka) [inline src/ios/ScanditSDK.mm]: isn't this more generic than the next if condition and so it'll always be text?

Comment (franek-scandit) [reply src/ios/ScanditSDK.mm]: It checks what type is `recognitionMode`, so if it's not a String (but Int) it'll execute 3rd or 4th block.

Comment (petradonka) [reply src/ios/ScanditSDK.mm]: true, nevermind, i'm not sure what i was thinking, it was obviously totally fine 😄

File: docs/Scandit.java [hunk 2]
```diff
@@ -719,11 +719,11 @@ public void show(function didScan, function didManualSearch=null, function didCa
             /**
              * Text recognition
              */
-            TEXT,
+            1,
             /**
              * Barcode/2d code recognition
              */
-            CODE
+            2
```
Comment (moritzhartmeier) [inline docs/Scandit.java]: These are the enums in the documentation, they should not change to numbers, the enums are still named TEXT and CODE.

Comment (franek-scandit) [reply docs/Scandit.java]: You're right. I'll fix this.

**Pull request merged**